app/models/system_notification.rb

```diff
@@ -7,4 +7,11 @@ class SystemNotification < ApplicationRecord
   validates :title, :modal_title, :modal_body, :description,
             :source_created_at, :source_id, :last_time_changed_at,
             presence: true
+
+  def self.last_sync_timestamp
+    SystemNotification
+      .order(last_time_changed_at: :desc)
+      .first&.last_time_changed_at&.to_i ||
+      User.order(created_at: :desc).first&.created_at&.to_i
```

Comment:
`&` operator is required here. 
We need it at `first&` ta avoid `.last_time_changed_at` or `.created_at` for `nil`, and before `.to_i`, to avoid convertig `nil` to `integer` becaseu 'nil.to_i = 0`

👍 for comment line

Reply:
Ah... And Why I am using integer here is, because we pass timestmap as URL query param to service...